projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/show-dialog.spec.tsx

```diff
@@ -52,23 +48,23 @@ describe('showRunJobDialog', () => {
       body: <RunJobDialog jobPath={jobData.get(VdkOption.PATH)!} />,
       buttons: [Dialog.okButton(), Dialog.cancelButton()]
     });
-    expect(alertMock as jest.Mock).toHaveBeenCalledWith(
-      'Job completed successfully!'
-    );
   });
 
   it('should call the jobRunRequest function if the user clicks the accept button', async () => {
-    const alertMock = jest
-      .spyOn(window, 'alert')
-      .mockImplementationOnce(() => {});
-
     // Call the function
     await showRunJobDialog();
 
     // Expect the jobRunRequest function to have been called
     expect(jobRunRequest).toHaveBeenCalled();
-    expect(alertMock as jest.Mock).toHaveBeenCalledWith(
-      'Job completed successfully!'
+    expect(showDialog).toHaveBeenCalledWith(
+      {
+        title: 'Run Job',
+            body: <div className='vdk-run-dialog-message-container'>
+            <p className='vdk-run-dialog-message'>Success!</p>
+            <span className='vdk-tick-element'>✔</span>
+          </div>,
```

Comment:
nit: I think that it might be a good idea to refactor the HTML code to semantic HTML. It would definitely affect the quality of the code in a good way. This is a more general topic; maybe we should discuss it elsewhere.